cc_proxy.go

```diff
@@ -205,10 +203,21 @@ func (p *ccProxy) sendCmd(cmd interface{}) (interface{}, error) {
 		return nil, fmt.Errorf("Wrong command type, should be hyperstartProxyCmd type")
 	}
 
-	err := p.client.Hyper(proxyCmd.cmd, proxyCmd.message)
+	data, err := json.Marshal(proxyCmd.message)
 	if err != nil {
 		return nil, err
 	}
 
+	var tokens []string
+	if proxyCmd.token == "" {
+		tokens = nil
+	} else {
+		tokens = append(tokens, proxyCmd.token)
+	}
+
+	if err := p.client.HyperWithTokens(proxyCmd.cmd, tokens, json.RawMessage(data)); err != nil {
```

Comment:
Good question... I cannot remember why we decided to return this interface initially. Let me check this and I will remove it if it is not needed.